ansible/bootstrap.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Bootstrap node servers
+  hosts: all
+  remote_user: master
+  become: yes
+
+  vars:
+    bootstrap_version: master
+    vault_role: ae-node
+    package: "https://s3.eu-central-1.amazonaws.com/aeternity-node-builds/aeternity-latest-ubuntu-x86_64.tar.gz"
+
+  tasks:
+    - name: Clone infrastructure repo
+      git:
+        repo:  https://github.com/aeternity/infrastructure.git
+        dest: /infrastructure
+        version: "{{ bootstrap_version }}"
+        depth: 1
+        force: yes
+
+    - name: Run bootstrap.sh
+      shell: |
+        /infrastructure/scripts/bootstrap.sh \
+         --vault_addr={{ vault_addr }} \
+         --vault_role={{ vault_role }} \
+         --env={{ env }} \
+         --epoch_package={{ package }} \
```

Comment:
rebase and sync the name with master

Reply:
fixed https://github.com/aeternity/infrastructure/pull/255/commits/4bee91dd652f35af5c5c3318671d9bbfcf9da9a8